server/webserver.go

```diff
@@ -132,10 +132,10 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 			pNames = append(pNames, p.Name)
 		}
 		config := struct {
-			BroadcastConfig    *BroadcastConfig
+			MaxPrice           *big.Rat
```

Comment:
We'll need to update the CLI wizard as well because it currently tries to unmarshal the JSON response into a struct with a `big.Int` (https://github.com/livepeer/go-livepeer/blob/master/cmd/livepeer_cli/wizard_stats.go#L363) which would fail at the moment.